Subject: fix soversion at 7
From: Michael Tokarev <mjt@tls.msk.ru>
Date: Mon, 31 Dec 2018 21:33:08 +0300

Unfortunatley upstream apparently can't learn, and increments
soversion with every new release, even if there's no reason to
and the new library is completely backwards-compatible.

Undo this nonsense and keep soversion at 7 for now.

--- a/lib/Makefile.am	2016-10-09 21:54:23.000000000 +0300
+++ b/lib/Makefile.am	2018-12-31 21:23:20.530483158 +0300
@@ -17 +17 @@
-SOCURRENT=9
+SOCURRENT=7
